Duplicati/Library/Backend/WEBDAV/WEBDAV.cs

```diff
@@ -311,7 +313,7 @@ public string Description
 
         public void Test()
         {
-            this.TestList();

```

Comment:
In principle, the call should be `this.TestList().FirstOrDefault()` to make sure the code runs, but all the other backends do it this way also (load all files).

Reply:
I only reverted that commit (38883285eee5598e37d56e6ddc417fad7d872ed3) as indicated in description